nilearn/maskers/nifti_spheres_masker.py

```diff
@@ -506,7 +506,9 @@ def _reporting(self):
             )
             regions_summary["relative size (in %)"].append("not implemented")
             if idx in spheres_to_be_displayed:
-                display = plotting.plot_img(img, cut_coords=positions[idx])
+                display = plotting.plot_img(
+                    img, cut_coords=positions[idx], cmap="grey"
```

Comment:
unrelated: changes the color map of the report to grey scale

Reply:
Is there a reason for that ?